python/pyspark/mllib/classification.py

```diff
@@ -326,55 +336,65 @@ def train(rdd, i):
 
 class LogisticRegressionWithLBFGS(object):
     """
+    Train a classification model for Multinomial/Binary Logistic Regression
```

Comment:
Here and similar contexts added minimal docs, so summary is actually visible in top level doc. Otherwise we just get version added comment, which is not very useful.